hexstody-hot/src/worker.rs

```diff
@@ -1,15 +1,55 @@
+use hexstody_api::types::ConfirmedWithdrawal;
 use hexstody_btc_api::events::*;
 use hexstody_btc_client::client::BtcClient;
 use hexstody_db::{
     state::State,
-    update::{btc::BestBtcBlock, StateUpdate, UpdateBody},
+    update::{btc::BestBtcBlock, StateUpdate, UpdateBody, results::UpdateResult},
 };
 use log::*;
 use std::sync::Arc;
 use std::time::Duration;
 use tokio::sync::{mpsc, Mutex};
 use tokio::time::sleep;
 
+pub async fn update_results_worker(
+    btc_client: BtcClient,
+    state_mx: Arc<Mutex<State>>,
+    mut update_receiver: mpsc::Receiver<UpdateResult>,
+){
+    trace!("Starting update results worker");
+    loop {
+        match update_receiver.recv().await {
+            Some(upd) => match upd {
+                UpdateResult::WithdrawConfirmed(id) => {
+                    let sreq = {   
+                        let state = state_mx.lock().await;
+                        state.get_withdrawal_request(id)
+                    };
+                    if let Some(req) = sreq {
+                        let confirmations = req.confirmations.iter().map(|wrd| wrd.clone().into()).collect();
+                        let rejections = req.rejections.iter().map(|wrd| wrd.clone().into()).collect();
+                        let cw = ConfirmedWithdrawal{
+                            id,
+                            user: req.user,
+                            address: req.address,
+                            created_at: req.created_at.to_string(),
```

Comment:
Change it to
`created_at: req.created_at.format("%Y-%m-%d %H:%M:%S").to_string(),`
Otherwise signatures won't match:
`"created_at":"2022-07-05 01:21:05.626709"`
vs
`"created_at":"2022-07-05 01:21:05"`